packages/docusaurus-plugin-content-blog/src/__tests__/blogUtils.test.ts

```diff
@@ -214,6 +257,7 @@ describe('linkify', () => {
         frontMatter: {},
         authors: [],
         formattedDate: '',
+        unlisted: false,
```

Comment:
This was added as I noticed a type error when I was in the file, so I thought I'd fix it!